core/src/components/datetime/datetime.tsx

```diff
@@ -2056,7 +2169,8 @@ export class Datetime implements ComponentInterface {
     const hasWheelVariant =
       (presentation === 'date' || presentation === 'date-time' || presentation === 'time-date') && preferWheel;
 
-    renderHiddenInput(true, el, name, value, disabled);
+    const formattedValue = Array.isArray(value) ? JSON.stringify(value) : value;
```

Comment:
~ Comparing against Select, the implementation abstracts this behavior to a `parseValue` function. Also interestingly it uses `.join(',')` instead of `JSON.stringify`, I could not find reasoning behind this.

It may make sense to make the implementations use similar/same language, so that when later working with these components they have the same implementation style.

Reply:
Huh, I'm not sure why we use `join` instead of `stringify` there but it would be good to have the two match up. 🤔 Otherwise, `datetime` will have the `[]` brackets around the values, while `select` doesn't. Good catch!

Reply:
Done 👍 I used `formatValue` instead of `parseValue` since `ion-datetime` tends to use `parse` language to signify manipulating date parts specifically.